sentry-core/src/clientoptions.rs

```diff
@@ -98,6 +99,16 @@ pub struct ClientOptions {
     pub trim_backtraces: bool,
     /// The user agent that should be reported.
     pub user_agent: Cow<'static, str>,
+    /// The sample rate for traces. (if set, must be inside the 0.0 - 1.0 range)
+    ///
+    /// Tracing is enabled if either this or `traces_sampler` is defined. If both
+    /// are defined, `traces_sample_rate` is ignored.
+    pub traces_sample_rate: Option<f32>,
```

Comment:
I’m thinking if we should rather default this to `0.0` instead of having it as `Option`.